manifests/init.pp

```diff
@@ -204,6 +210,8 @@
   Integer[0] $content_service_worker_timeout = 90,
   Integer[0] $api_service_worker_timeout = 90,
   Hash[String[1], String[1]] $api_client_auth_cn_map = {},
+  Boolean $cache_enabled = false,
+  Variant[Integer[0], Enum['None']] $cache_expires_ttl = 60,
```

Comment:
Should we make this an optional parameter? Then if you don't set this, the whole `CACHE_SETTINGS` will not be included in `settings.py` and it'll use the application built in defaults.

I also wonder what `None` means. No TTL expiration and thus infinite caching?

Reply:
Correct on the None front, found that in their docs. I'll make this optional.

Reply:
We can include the `Use None for infinite caching` in our doc string.